configure.py

```diff
@@ -416,19 +434,11 @@
             Object(Matching, "zlib/adler32.c"),
             Object(Matching, "zlib/crc32.c"),
             Object(Matching, "zlib/deflate.c"),
+            Object(Matching, "zlib/trees.c"),
+            Object(Matching, "zlib/zutil.c"),
             Object(Matching, "zlib/inflate.c"),
+            Object(NonMatching, "zlib/inftrees.c"),
```

Comment:
building this with a different compiler was intentional

whats the rationale behind this

Reply:
I just traced it down and it looks like Wii 1.0 got messed up in the current compilers.zip, will fix

Reply:
Fixed by using 1.0a